.github/workflows/reusable-docker-build-publish.yml

```diff
@@ -0,0 +1,349 @@
+        default: "true"
+      docker-tag-custom-suffix:
+        description: |
+          This is an optional input that can be used to add a custom suffix to
+          the docker tag. Useful to label different images sharing the same AWS
+          ECR repository. Recommended to prefix input string with a hyphen "-".
+
+          Examples:
+            - "-ccip"
+            - "-plugins"
+        required: false
+        type: string
+      git-sha:
+        description: |
+          The full git SHA to build from.
+
+          Example: 604b8b878253b4cce2d0dba37b53644b67abbf97
+        required: true
+        type: string
+      github-event-name:
+        description: |
+          > The name of the event that triggered the workflow run.
+
+          Examples:
+            - pull_request
+            - push
+            - schedule
+        required: true
+        type: string
+      github-ref-name:
+        description: |
+          > The short ref name of the branch or tag that triggered the workflow
+          > run. This value matches the branch or tag name shown on GitHub.
+
+          Examples:
+            - <pr_number>/merge # branch on pull_request events
+            - feature-branch-1  # branch on push events
+            - v1.0.0            # tag on push events
+        required: true
+        type: string
+      github-runner-amd64:
+        description: |
+          The GitHub Actions runner name for builds on the platform: amd64
+        required: false
+        type: string
+        default: "ubuntu-24.04"
+      github-runner-arm64:
+        description: |
+          The GitHub Actions runner name for builds on the platform: arm64
+        required: false
+        type: string
+        # NOTE: This runner is not yet available on non-public repos.
+        # Will require an override for private/internal repos.
+        default: "ubuntu-24.04-arm"
+      github-workflow-repository:
+        description: |
+          The repository name of the workflow that triggered the workflow run.
+
+          Example: smartcontractkit/chainlink
+        required: true
+        type: string
+
+    outputs:
+      docker-image-sha-digest-amd64:
+        description: "Docker image SHA digest for amd64 architecture"
+        value: ${{ jobs.build-publish.outputs.docker-image-sha-digest-amd64 }}
+      docker-image-sha-digest-arm64:
+        description: "Docker image SHA digest for arm64 architecture"
+        value: ${{ jobs.build-publish.outputs.docker-image-sha-digest-arm64 }}
+      docker-manifest-digest:
+        description: "Docker manifest digest"
+        value: ${{ jobs.docker-manifest.outputs.digest }}
+      docker-manifest-name:
+        description: "Docker manifest name"
+        value: ${{ jobs.docker-manifest.outputs.name }}
+      docker-manifest-tag:
+        description: "Docker manifest tag"
+        value: ${{ jobs.docker-manifest.outputs.tag }}
+jobs:
+  init:
+    name: init
+    runs-on: ubuntu-24.04
```

Comment:
should use inputted runner?

Reply:
Good thought. Updated in 84ad689cb10c6def53fde84fe836547377f26c80